test/test_configs/config.fixtures_mysql.test.yml

```diff
@@ -5,7 +5,7 @@ app_name: fixture-loader
 
 db_drivername: "mysql"
 db_hostname: "172.17.0.1"
```

Comment:
`GOSOLINE_MYSQL_MYSQL_HOST` ?

Reply:
i would say, we do this when we need it, so with #225